app/views/observations/show/_observation_details.html.erb

```diff
@@ -4,12 +4,12 @@
 
 <%= observation_details_when_where_who(obs: obs) %>
 
-<%= if @user
-  tag.div(class: "obs-projects", id: "observation_projects") do
-    obs.projects.each do |project|
-      tag.p("#{:PROJECT.t}: #{link_to_object(project)}")
-    end
-  end
+<% if @user
+  obs.projects.each do |project| %>
+    <%= "#{:PROJECT.t}: " %>
+    <%= link_to_object(project) %>
+    <br/>
+  <% end
```

Comment:
A big improvement because it actually works.
But I wish I could figure out the syntax for:
- including blocks wtihin `tag`s,
- including interpolated `link_to_object` in a string in a tag.
Maybe there isn't one.
@nimmo?

Reply:
@JoeCohen I'm not sure i've ever gotten a link interpolated in a string to concat right, but i have thrashed through hundreds of views and maybe there's a solution in there somewhere. But - see suggestion below.